<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="RoutesBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>271</width>
      <height>841</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="RoutesLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Маршруты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DumpButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>740</y>
      <width>171</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить как</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>820</y>
      <width>133</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>780</y>
      <width>133</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
